lib/tasks/impossible_overdue_appointments.rake

```diff
@@ -0,0 +1,47 @@
+namespace :impossible_overdue_appointments do
+  desc 'Set all appointments with a newer blood pressure reading than the created_at date to "visited"'
+  task :fix_impossible_overdue_appointments, [:user_id] => :environment do |_t, args|
+    card_reader_bot = User.find(args.user_id)
+    patients_created_by_bot = Patient.where(registration_user_id: card_reader_bot.id)
```

Comment:
```suggestion
    patients_created_by_user = Patient.where(registration_user_id: card_reader_bot.id)
```

Reply:
Changed var name to `patients_created_by_user`.